src/components/Pages/Back.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Back = () => (
+  <Link to="/">Back</Link>
+);
```

Comment:
- [ OPTIONAL] Great job on this :+1: but what if you use an `icon` to get back to the home page instead of a simple `back` word 
- eg : `<Link to="/">Icon</Link>`